packages/hs-react-ui/src/components/Modal/Modal.tsx

```diff
@@ -34,7 +36,7 @@ const ModalUnderlay = styled(Div)<{ backgroundBlur: string; backgroundDarkness:
 
     z-index: 1000;
 
-    backdrop-filter: blur(${backgroundBlur}) opacity(${1 - backgroundDarkness / 100});
+    backdrop-filter: blur(${backgroundBlur}) brightness(${backgroundDarkness});
```

Comment:
darkness is the inverse of brightness, so darkness 1 sounds like MAX darkness, but it's actually the opposite.

You could rename the prop to background brightness, or just subtract by 1 and keep the prop name (which is my suggestion)
```suggestion
    backdrop-filter: blur(${backgroundBlur}) brightness(${1 - backgroundDarkness});
```